module/batch-action/src/Infrastructure/JMS/Serializer/Handler/BatchActionActionHandler.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Infrastructure\JMS\Serializer\Handler;
+
+use Ergonode\BatchAction\Domain\ValueObject\BatchActionAction;
+use JMS\Serializer\Context;
+use JMS\Serializer\GraphNavigatorInterface;
+use JMS\Serializer\Handler\SubscribingHandlerInterface;
+use JMS\Serializer\Visitor\DeserializationVisitorInterface;
+use JMS\Serializer\Visitor\SerializationVisitorInterface;
+
+class BatchActionActionHandler implements SubscribingHandlerInterface
```

Comment:
Are those classes necessary? It seems pretty straightforward as a generic operation